.github/CONTRIBUTING.md

```diff
@@ -57,6 +57,7 @@ Follow these steps to backport a change into a release branch in compliance with
 3. Create a branch for your change, e.g. `git checkout -b backport`.
 4. When the reason to backport is not obvious from the original commit message, use `git cherry-pick -xe <original commit>` and add a reason. Otherwise use `git cherry-pick -x <original commit>`. That's fine for minor version updates that only include security and bug fixes, commits that fixes an otherwise broken package or similar. Please also ensure the commits exists on the master branch; in the case of squashed or rebased merges, the commit hash will change and the new commits can be found in the merge message at the bottom of the master pull request.
 5. Push to GitHub and open a backport pull request. Make sure to select the release branch (e.g. `release-20.09`) as the target branch of the pull request, and link to the pull request in which the original change was comitted to `master`. The pull request title should be the commit title with the release version as prefix, e.g. `[20.09]`.
+6. When the backport pull request is merged, reference the cherry-picked commit in the original pull request. If you have the privileges you can also remove the label `9.needs: port to stable` and replace it with `8.has: port to stable`. This way maintainers can keep track of missing backports easier.
```

Comment:
> When the backport pull request is merged, reference the cherry-picked commit in the original pull request.

I wouldn't want to put that extra work on someone. Just put the #XXX of the original PR in the backported PR.

> If you have the privileges you can also remove the label `9.needs: port to stable` and replace it with `8.has: port to stable`. This way maintainers can keep track of missing backports easier.
 
Ideally this would be done by a bot because this is easily automatable.

Also 

> If you have the permissions you can also remove the label `9.needs: port to stable` and replace it with `8.has: port to stable`. This way maintainers can keep track of missing backports easier.

Reply:
> I wouldn't want to put that extra work on someone. Just put the #XXX of the original PR in the backported PR.

Agreed.

> Ideally this would be done by a bot because this is easily automatable.

I'm not so sure about this. Matching PRs with their backports doesn't seem trivial to me. E.g. people tend to not write the exact `[20.09]` in the PR title and sometimes backport PRs address multiple PRs to unstable. Could we keep this part until a bot is written, that solves this problem?